__mocks__/swrv.js

```diff
@@ -3,12 +3,8 @@ const baseURL = import.meta.env.VITE_TRACTION_BASE_URL
 const flags = {
   flipper_id: 'User',
   features: {
-    dpl_277_enable_general_reception: { enabled: true },
-    dpl_478_enable_qc_results_upload: { enabled: true },
     enable_custom_select: { enabled: true },
```

Comment:
I am not sure whether we want to keep these flags or not. @seenanair any thoughts about when we should remove the custom component flags?

Reply:
I would say remove them. They are all in production.